.gitignore

```diff
@@ -1 +1,2 @@
 /target
+/src/data
```

Comment:
I guess that this can be removed, right?

Reply:
Yes it's leftover! removed at [c8a2d7f](https://github.com/summa-dev/summa-aggregation/pull/3/commits/c8a2d7f0ae2411b5e44a86837bc43dda5256f6be)